src/main.rs

```diff
@@ -21,6 +22,16 @@ enum Command {
     Serve(Serve),
 }
 
+pub fn create_runtime_tokio() -> Runtime {
```

Comment:
i'm slightly confused about this function- and then the contents of this function are repeated in the main function. can you tell me more? or we can hop on a call

Reply:
also you can probably skip the assignment here (remove the `let runtime` and `runtime`)

Reply:
that was my comment in the top, when using this function in main the remote assets broke, genuinly don't understand